app/scripts/modules/amazon/src/instance/details/instance.details.controller.js

```diff
@@ -70,6 +71,11 @@ module.exports = angular
         var displayableMetrics = instance.health.filter(function(metric) {
           return metric.type !== 'Amazon' || metric.state !== 'Unknown';
         });
+
+        // augment with target group healthcheck data
+        const targetGroups = getAllTargetGroups(app.loadBalancers.data);
+        getTargetGroupHealthCheckInfo(displayableMetrics, targetGroups);
```

Comment:
if we are going to mutate data in a function, I'd like the function name to indicate this.  I was confused because there was a `get` method but I didn't see the return value used.  
Perhaps something like `applyHealthcheckInfoToTargetGroups`?